kolibri/plugins/coach/assets/src/views/elapsed-time/index.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+
+  <span>{{ $tr($trUnit, {n: display}) }}</span>
```

Comment:
For devs, I think putting logic in javascript rather than ICU Message syntax when possible is easier to reason about. On the other hand for translators, putting it all in ICU syntax might (?) help them write more consistently. Not sure .. I don't think there's a clearly correct answer here.